examples/tokengating-cart-checkout-validation/README.md

```diff
@@ -0,0 +1,41 @@
+# Shopify Function development with Rust
```

Comment:
Does this run automatically? I read recently here that we need the `checkoutServerExtension` piece in step 2.4 for here: https://shopify.dev/docs/apps/checkout/validation/cart-checkout#step-2-activate-the-cart-and-checkout-validation-with-graphiql